sky/packages/sky/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui' as ui;
+
+/// Dart-layer version of ui.Event
+class InputEvent {
+
+  InputEvent({ this.type }) : timeStamp = 0.0;
+
+  factory InputEvent.fromSkyEvent(ui.Event event) {
```

Comment:
I did s/sky/ui further down, I missed this one.

I wonder if s/sky/native here would be clearer?